cirq-core/cirq/ops/raw_types.py

```diff
@@ -277,6 +280,16 @@ def on_each(self, *targets: Union[Qid, Iterable[Any]]) -> List['cirq.Operation']
     def wrap_in_linear_combination(
         self, coefficient: Union[complex, float, int] = 1
     ) -> 'cirq.LinearCombinationOfGates':
+        """Return a LinearCombinationOfGates with this gate.
```

Comment:
nit: The other docstrings use third person singular rather than imperative form, so maybe let's change to "Returns" for consistency?

(Applies to `with_probability` below, too.)

Reply:
Done.